src/ATen/THLongStorageView.h

```diff
@@ -4,33 +4,49 @@
 
 namespace at {
 
+static inline bool is_noelem_tensor_size(ArrayRef<int64_t> size) {
```

Comment:
Can you add a doc comment about what this function is for? i.e. why does

```
size.size() == 1 && size[0] == 0;
```

represent a no element tensor under our current confusing view of Tensor, Undefined Tensor, Scalar etc.

Reply:
sure.